### PR TITLE
WIM review items for beta

### PR DESCRIPTION
**Pull request by tgross-usgs**
Fixes the following issues.

- #272 unhides the report text input until print
- #270 couldn't replicate this issue, the identifier shows up fine for me in all browsers
- #278 displays version in About modal
- #277 kudos to projects in About modal, also adds ToT application description
- #269 adds google analytics

### CODE REVIEW

File: src/app/components/about/about.component.html [hunk 1]
```diff
@@ -2,24 +2,25 @@
 	<div class="modal-box">
 		<!-- Header -->
 		<div class="modal-top">
-			<div class="title">About Time of Travel</div>
+			<div class="title">About the Time of Travel application v0.5.0</div>
```
Comment (amedenblik) [inline src/app/components/about/about.component.html]: Consider dynamically loading the version number from the [config.json (line 2)](https://github.com/USGS-WiM/TravelTime/blob/dev/dist/assets/data/config.json#L2).

File: src/app/components/about/about.component.html [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
-				the authorized or unauthorized use of the software.
-			</p>
+			<p>The Time of Travel application estimates travel times and concentrations between a user-selected spill location and locations of downstream critical infrastructure, such as municipal water intakes based on a user-specified distance. Estimates of travel time are provided for the leading edge, peak, and trailing edge, and peak concentration will be provided under the most-probable velocity and the maximum-probable velocity of the spilled oil, assuming the flow is steady and the spill is instantaneously and completely mixed. Users are required to input the mass of the spilled contaminant and the decay rate to obtain estimates of concentrations. Users also are able to search the stream network upstream from a selected location and identify potential spill travel times within a specified distance. 
+
+			<h4>Behind the scenes...</h4>
+			<p>The Time of Travel application utilizes <a href="https://nhdplus.com/NHDPlus/" target="_blank">NHDPlus Version 2</a> for it's data network and is built with <a href="https://angular.io/" target="_blank">Angular</a>, <a
```
Comment (amedenblik) [inline src/app/components/about/about.component.html]: "it's" should be "its"

Comment (tgross-usgs) [reply src/app/components/about/about.component.html]: Lol! Good catch. I copied this right out of Pete's proposal doc and didn't proof read what I didn't change. Oops!

File: src/app/components/about/about.component.html (hunk 1 above)
Comment (tgross-usgs) [reply src/app/components/about/about.component.html]: Great idea. Wish I'd thought of it. ;-)

**Review by amedenblik — CHANGES_REQUESTED** [review]
Issue #272:

- ✔ Confirmed that report title has a text box for input 
- ✔ Confirmed that text box content propagates to the printable report 
- ❌ Noticed that printable report does not show all elements? I selected "☑ Map of study area" and "☑ Graph of timeline" in Step 2 of Build Report, but the map and graph of timeline don't show up when I click "Print". Example:
image

Issue #270:

- ✔ Confirmed that the USGS logo shows up in Chrome, Firefox, and Edge. Could not recreate the issue.
- ✔ Confirmed that an alt tag was added to the logo 

Issue #278 

- ✔ Confirmed that the version number shows up in the About section

Issue #277

- ✔ Confirmed that kudos to other projects are listed
- ✔ Confirmed presence of application description
- ❌ "it's" typo is still present

Issue #269 

- ✔ Confirmed Google Analytics 

Other comments:

- I had issues with installing and running the application. Some changes may need to be made. The following steps were necessary to get the application to run on my machine:
```
git clone https://github.com/USGS-WiM/TravelTime.git
cd TravelTime
nvm use 12.14
npm install
npm uninstall node-sass
npm install node-sass@4.12
ng serve -o
```

**Comment by tgross-usgs** [comment]
@amedenblik The map and charts now display in the print preview thanks to @mitchas. I noticed that the lat lon for mouse position weren't working in the report so I removed the section altogether along with the map zoom since it isn't needed in the report. The typo in the About section should now be fixed as well. (I'd forgotten to save my change and the app doesn't alert me about unsaved changes.)

**Review by amedenblik — APPROVED** [review]

**Pull request merged**